grid_map/src/error.rs

```diff
@@ -10,6 +10,8 @@ pub enum Error {
     ImageError(#[from] image::ImageError),
     #[error("yaml scan error: {0}")]
     YamlScanError(#[from] yaml_rust::ScanError),
```

Comment:
It does not need to be part of this PR, but it would be nice if other `yaml_rust` uses could also be replaced by `serde_yaml`.